.github/workflows/python-app.yml

```diff
@@ -72,7 +71,10 @@ jobs:
         black usl_pipeline/map_uploader --check
     - name: Test with pytest
       run: |
-        pytest usl_pipeline/map_uploader
+        pytest usl_pipeline/map_uploader usl_pipeline/usl_lib
+    - name: MyPy Type Checking
+      run: |
+        mypy usl_pipeline/map_uploader usl_pipeline/usl_lib
```

Comment:
Can you do separate checks like
```
mypy usl_pipeline/map_uploader
mypy usl_pipeline/usl_lib
```
(otherwise I have mypy error like in https://github.com/UrbanSystemsLab/climateiq-cnn/actions/runs/8695726630/job/23847381930?pr=25)?

Reply:
Done. We don't really need to MyPy both anyway here.